sawtooth-daml-rpc/src/main/java/com/blockchaintp/sawtooth/daml/rpc/SawtoothReadService.java

```diff
@@ -20,13 +20,11 @@
 import com.blockchaintp.sawtooth.daml.protobuf.ConfigurationMap;
 import com.blockchaintp.sawtooth.daml.rpc.events.DamlLogEventHandler;
 import com.blockchaintp.sawtooth.daml.util.Namespace;
-import com.daml.ledger.participant.state.backport.TimeModel;
-import com.daml.ledger.participant.state.v1.Configuration;
-import com.daml.ledger.participant.state.v1.LedgerInitialConditions;
-import com.daml.ledger.participant.state.v1.Offset;
-import com.daml.ledger.participant.state.v1.ReadService;
-import com.daml.ledger.participant.state.v1.Update;
+import com.daml.ledger.participant.state.v1.*;
```

Comment:
Don't import * unless you are in fact using the majority ( 8+ as a rule of thumb) of a given package, pollutes the namespace.  You've basically done the scala equivalent elsewhere in this same pull , so I'm guessing you get that :)

Reply:
sorry this was intellij being overzealous on import optimisation for me -- now made explicit